build/config.gypi

```diff
@@ -0,0 +1,373 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
This file should be added. Not sure why it was generated but it should be removed.